components/brave_ads/core/internal/account/user_data/fixed/page_land_user_data_unittest.cc

```diff
@@ -20,34 +21,37 @@ class BraveAdsPageLandUserDataTest : public test::TestBase {};
 TEST_F(BraveAdsPageLandUserDataTest,
        BuildPageLandUserDataForHttpResponseStatusErrorPage) {
   // Act
-  const base::Value::Dict user_data = BuildPageLandUserData(
-      TabInfo{/*id=*/1,
-              /*is_visible=*/true,
-              /*redirect_chain=*/{GURL("https://brave.com")},
-              /*is_error_page=*/true,
-              /*is_playing_media=*/false});
+  const base::Value::Dict user_data = BuildPageLandUserData(TabInfo{
+      /*id=*/1,
+      /*is_visible=*/true,
+      /*redirect_chain=*/{GURL("https://brave.com")}, net::HTTP_NOT_FOUND,
+      /*is_playing_media=*/false});
 
   // Assert
   EXPECT_EQ(base::test::ParseJsonDict(
                 R"(
                     {
-                      "httpResponseStatus": "errorPage"
+                      "httpResponseStatus": "404"
```

Comment:
maybe we can add tests for 2xx and 3xx code classes as well?

Reply:
Sounds good. Added tests for all status code classes.